libs/core/src/components/sage-link/sage-link.tsx

```diff
@@ -0,0 +1,58 @@
+import { Component, h, Prop } from '@stencil/core';
+
+/**
+ * @slot - Content is placed between the opening closing tags
+ */
+@Component({
+  tag: 'sage-link',
+  styleUrl: 'sage-link.scss',
+  shadow: true,
+})
+export class SageLink {
+  /**
+   * When enabled, opens link in a new tab.
+   * @defaultValue false
+   */
+  @Prop() external = false;
+
+  /**
+   *
+   * Modifies the look of the link
+   */
+  @Prop() variant: 'inline' | 'plain' = 'inline';
+
+  /**
+   *
+   * The Font size follows t-shirt model
+   * sm: 12px
+   * md: 14px
+   * lg: 16px
+   * @defaultValue lg
+   */
+  @Prop() fontSize: 'sm' | 'md' | 'lg' = 'lg';
+
+  /**
+   * The URL that the hyperlink points to.
+   */
+  @Prop() href!: string;
+
+  render() {
+    const externalIcon = (
+      <svg role="presentation" width="15" height="14" viewBox="0 0 15 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path
+          fill-rule="evenodd"
```

Comment:
We can likely clean up this SVG - fill-rules and clip-rules usually aren't needed. Further, we could do some fill cleanup etc...